airbyte-webapp/src/config/config.ts

```diff
@@ -0,0 +1,36 @@
+import { AirbyteWebappConfig } from "./types";
+
+export const config: AirbyteWebappConfig = {
+  segment: {
+    token: window.SEGMENT_TOKEN ?? process.env.REACT_APP_SEGMENT_TOKEN,
+    enabled: window.TRACKING_STRATEGY === "segment",
+  },
+  apiUrl:
+    window.API_URL ??
+    process.env.REACT_APP_API_URL ??
+    `${window.location.protocol}//${window.location.hostname}:8001/api`,
+  connectorBuilderApiUrl:
+    process.env.REACT_APP_CONNECTOR_BUILDER_API_URL ?? `${window.location.protocol}//${window.location.hostname}:8003`,
+  healthCheckInterval: 20000,
```

Comment:
Given that this is never actually configurable, I suggest we just move that as a constant into the `HealthPollService.tsx`